lcoe_calculator_documentation.html

```diff
@@ -38,33 +38,35 @@ <h5 class='card-header'>Example: Cell cost reduction</h5>
           </div>
         </div>
       </p>
       <h2>Presets</h2>
-      <p>The calculator includes hundreds of presets for the input parameters. To replace the <strong>baseline</strong> input parameters with a set of preset parameters, use the <strong>presets</strong> button. To update the <strong>proposed</strong> inputs to match the preset, use the <strong>copy from baseline</strong> button after choosing a preset.</p>
-      <p>Each preset represents a simulated PV system with a particular <strong>cell technology</strong>, module <strong>package type</strong>, <strong>system type</strong>, and geographic <strong>location</strong>. Input values for module cost come from NREL's benchmark module cost studies. Energy yield has been simulated using SAM with TMY inputs for each location. Silicon modules are simulated with an anti-reflective coating (ARC) on the glass.</p>
+      <p>The calculator includes hundreds of presets for the input parameters. Choose settings in the <strong>presets for inputs</strong> section and then press <strong>apply to baseline</strong> or <strong>apply to proposed</strong> to load the preset parameter values.</p>
+      <p>Each preset represents a simulated PV system with a particular <strong>cell technology</strong>, module <strong>package type</strong>, <strong>system type</strong>, and geographic <strong>location</strong>. Input values for module cost come from NREL's benchmark module cost studies. Energy yield has been simulated using <a href="https://sam.nrel.gov/">SAM</a> with <a href="https://nsrdb.nrel.gov/">NSRDB</a> weather data for each location. More details on this simulation are available in the <a href="https://github.com/NREL/PVLCOE/">repository documentation</a>.</p>
       <dl class='row'>
         <dt class='col-sm-3'>Cell technology</dt>
-        <dd class='col-sm-9'>Cell technology can be monocrystalline silicon (mono-Si), multicrystalline silicon (multi-Si), or cadmium telluride (CdTe). Cell technology affects <strong>cell cost</strong>, <strong>efficiency</strong>, <strong>energy yield</strong>, <strong>degradation rate</strong>, <strong>BOS cost</strong>, and the available values for <strong>package type</strong> and <strong>system type</strong>.</dd>
+        <dd class='col-sm-9'>Cell technology can be monocrystalline silicon (mono-Si), multicrystalline silicon (multi-Si), or cadmium telluride (CdTe). Modules with silicon cell technology are simulated with an anti-reflective coating (ARC) on the glass. Cell technology affects <strong>cell cost</strong>, <strong>efficiency</strong>, <strong>BOS cost</strong>, and the available values for <strong>package type</strong> and <strong>system type</strong>.</dd>
```

Comment:
Cell technology affects energy yield in the case of SAT
```suggestion
        <dd class='col-sm-9'>Cell technology can be monocrystalline silicon (mono-Si), multicrystalline silicon (multi-Si), or cadmium telluride (CdTe). Modules with silicon cell technology are simulated with an anti-reflective coating (ARC) on the glass. Cell technology affects <strong>cell cost</strong>, <strong>efficiency</strong>, <strong>energy yield</strong>, <strong>BOS cost</strong>, and the available values for <strong>package type</strong> and <strong>system type</strong>.</dd>
```